gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```

Comment:
In the build.gradle file you are using version 7.0.3 of the gradle plugin, maybe we can use the same in here.

Reply:
Funny thing: 7.0.3-all does not exist, build fails. So for now we will keep as it is.